DigitalOffice/Services/AuthenticationService/Broker/Senders/AuthenticationSender.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using MassTransit;
+using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.InternalModels.BrokerInterface;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders.Interface;
+
+namespace LT.DigitalOffice.AuthenticationService.Broker.Senders
+{
+    public class AuthenticationSender : ISender<string, UserCredentialsInternalRequest>
+    {
+        private readonly IRequestClient<UserCredentialsInternalRequest> requestClient;
+
+        public AuthenticationSender([FromServices] IRequestClient<UserCredentialsInternalRequest> requestClient)
+        {
+            this.requestClient = requestClient;
+        }
+
+        public async Task<Response<IOperationResult<UserCredentialsInternalRequest>>>
+            GetResponseFromBroker(string userEmail)
+        {
+            var request = new
+            {
+                Email = userEmail
+            };
+
+            var response = await requestClient.GetResponse<IOperationResult<UserCredentialsInternalRequest>>(request);
```

Comment:
I think it should response something other. Not UserCredentialsInternalRequest

Reply:
Fixed, i added new response model for broker